src/starlite_saqlalchemy/service/sqlalchemy.py

```diff
@@ -122,3 +122,24 @@ async def new(cls: type[RepoServiceT]) -> AsyncIterator[RepoServiceT]:
         """
         async with async_session_factory() as session:
             yield cls(session=session)
+
+
+class PaginatedRepositoryService(RepositoryService):
```

Comment:
While we are still breaking things, I'd rather just change this on `RepositoryService`.